services/task_service.py

```diff
@@ -90,12 +90,14 @@ def update_task():
         chosen_column_name, f"\nEnter the new {chosen_column_name}: "
     )
 
+
+def update_column_value(id, column, newValue):
```

Comment:
I created here a separate function to handle the actual change on the DB the client logic still stays on the function from above.